services/movie.py

```diff
@@ -0,0 +1,40 @@
+from django.db.models import QuerySet
+
+import init_django_orm  # noqa: F401
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids
+        ).filter(actors__id__in=actors_ids)
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
The `get` method will raise a `Movie.DoesNotExist` exception if no movie with the given `id` is found. Consider handling this exception to avoid potential runtime errors.

Reply:
We expect the id to be in the database

Reply:
Just use get_or_404